src/test/java/posters/pageobjects/pages/checkout/ShippingAddressPage.java

```diff
@@ -37,9 +37,10 @@
 
     @Override
     @Step("ensure this is a shipping address page")
-    public void isExpectedPage()
+    public ShippingAddressPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.